tokio-trace-beacon/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+//! This library emits instrumentation points akin to those seen in real applications.
+//! Its main intended use case is debugging subscribers.
+
+#[macro_use]
+extern crate tokio_trace;
+
+mod server;
+mod yak_shave;
+
+pub enum ApplicationKind {
```

Comment:
What do you think about making this a trait with a `fn emit(&self)`, instead? Using an enum means that any time we add new variants, that's a breaking change on any subscribers using this in their tests.